### DISCUSSION TITLE
Android Development Programming Language

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#64332] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Which language should I used for Android Development and if possible, please recommend any resource.

**Comment by Halle-Group** [comment]
The primary language used for Android app development is Java. However, Kotlin has become an increasingly popular choice for Android development. Kotlin is a programming language that can be used in place of Java but still runs on the Java Virtual Machine (JVM). Here are some resources that you can use to learn either Java or Kotlin for Android development:

1. Java:
   - Oracle Java Tutorials: https://docs.oracle.com/javase/tutorial/
   - Udacity's "Become an Android Developer" Course: https://www.udacity.com/course/android-developer-nanodegree--nd801
   - Coursera's "Java for Android" Course: https://www.coursera.org/learn/java-for-android

2. Kotlin:
   - Kotlin Official Documentation: https://kotlinlang.org/docs/reference/android-overview.html
   - Android Kotlin Fundamentals Course: https://developer.android.com/courses/kotlin-android-fundamentals/overview
   - Udacity's "Kotlin for Android Developers" Course: https://www.udacity.com/course/kotlin-for-android-developers--ud888

Keep in mind that learning the programming language is necessary but not sufficient for Android development. You will also need to learn how to use Android Studio, the official Integrated Development Environment (IDE) for Android app development. Thus, you should also consider exploring online tutorials and courses for Android Studio.

**Comment by mdxabu** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> Which language should I used for Android Development and if possible, please recommend any resource.

@iLakshya , There are two languages used for android development [ Java and Kotlin ] 
But now kotlin is in the trend , so kotlin is kinda good
Kotlin has gained popularity in recent years and is now used by major companies such as Google, Netflix, Amazon, and Twitter.
